packages/ignition/src/app/[lang]/(content)/opengraph/[...collection]/route.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @next/next/no-img-element */
+
+import OpenGraph from "@/components/opengraph";
+import { Languages } from "@/types";
+import { NextRequest } from "next/server";
+import * as changeCase from "change-case";
+import { IconsManifest } from "rocketicons/data";
+
+const loader = new Map([
```

Comment:
We should move it to the static-generator and importing from there, as we use on icon ensuring all changes on the icons package will reflect here